### DISCUSSION TITLE
is uRun possible with adblock ?

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1076] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
My school does not allow [uBlock origin](https://chromewebstore.google.com/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm?hl=en) but they allow [adBlock](https://chromewebstore.google.com/detail/adblock-%E2%80%94-le-meilleur-blo/gighmmpiobklfepjocnamgkkbiglidom).
Is an exploit like uRun can work with adBlock ?

**Comment by The-bigMan69** [comment]
ublock is a ad blocker

**Comment by ghost** [comment]
no

**Reply by ghost** [reply]
nope

**Reply by SIMONHSCH** [reply]
scroll down on [this page](https://tinyurl.com/adblockcustomfilter) if you have adblock

**Comment by 3kh0** [comment, marked as answer]
No, ask your local IT admin to use ublock instead, its open source :D

**Reply by Inglan2** [reply]
> how to do that ?

go to your admin say something like: "theres this great adblocker called ublock that blocks basically all ads and trackers including youtube ads and its open source can you unblock it?"

**Reply by AshtonDavies** [reply]
The fact that some (few) actually accept these requests is crazy.

**Reply by SIMONHSCH** [reply]
The question is not how to ask but how to contact the admin

**Reply by Inglan2** [reply]
ask a teacher. the teacher could maybe even send an email. or talk to it at your school

**Reply by SIMONHSCH** [reply]
But I discovered that on adblock settings when u go on Filter List tab we can add custom filters with URLs that can run scripts
Example:
when u watch a video on [youtube](YouTube.com), on the adBlock pop-up,there is a button that can run scripts and allow ads on some YouTube channels.

**Reply by Inglan2** [reply]
Hmm interesting. I'll look into it

**Reply by SIMONHSCH** [reply]
did you look ?

**Reply by Inglan2** [reply]
no couldnt find anything

**Comment by ghost** [comment]
yes 3kh0!
